src/HDInsight/HDInsight/ManagementCommands/GetAzureHDInsightAzureMonitorCommand.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.HDInsight.Commands;
+using Microsoft.Azure.Commands.HDInsight.Models;
+using Microsoft.Azure.Commands.HDInsight.Models.Management;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.HDInsight
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightAzureMonitor", DefaultParameterSetName = SetByNameParameterSet)]
+    [OutputType(typeof(AzureHDInsightAzureMonitor))]
+    public class GetAzureHDInsightAzureMonitorCommand : HDInsightCmdletBase
+    {
+        #region Input Parameter Definitions
```

Comment:
My proposal is to remove all the prefixes "Set" as you cmdlet verb is not "Set".
```
        private const string ByNameParameterSet = "ByNameParameterSet";
        private const string ByResourceIdParameterSet = "ByResourceIdParameterSet";
        private const string ByInputObjectParameterSet = "ByInputObjectParameterSet";
```

Reply:
> 

fixed. Thanks~